notebook/notebookapp.py

```diff
@@ -226,7 +226,7 @@ def init_settings(self, jupyter_app, kernel_manager, contents_manager,
             },
             version_hash=version_hash,
             ignore_minified_js=jupyter_app.ignore_minified_js,
-            
```

Comment:
Could we ask you to redo this with your editor set not to strip whitespace? Sorry for the hassle, but the changes can cause unnecessary merge conflicts, and make tools like `git blame` less useful.

Other than this, it looks great, thanks :-)